terraform/Readme.md

```diff
@@ -4,7 +4,7 @@ cd terraform/
 # initialize working directory:
 terraform init
 # provision the droplets and run the experiments:
-terraform apply -var="DO_API_TOKEN=use_your_own_do_token" -var="SSH_KEY_FILE=$HOME/.ssh/id_rsa.pub" -var="pvt_key=$HOME/.ssh/id_rsa" -var="NODES=10" -var="ROUNDS=25" -var="NUM_doctlLEAVES=32"
+terraform apply -var="DO_API_TOKEN=use_your_own_do_token" -var="SSH_KEY_FILE=$HOME/.ssh/id_rsa.pub" -var="pvt_key=$HOME/.ssh/id_rsa" -var="NODES=10" -var="ROUNDS=25" -var="NUM_LEAVES=32"
```

Comment:
While [doctl](https://github.com/digitalocean/doctl) is quite useful -e.g. to destroy the droplets in case there was an error and terraform destroy doesn't do the job in that case - it's certainly not useful here :-D 

Thanks for fixing this @evan-forbes!